applications/zDebugger/zDebugger.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
```

Comment:
Code should have license headers like this one:
https://github.com/overte-org/community-apps/blob/master/applications/aaswitcher/app-aaSwitcher.js